### ISSUE TITLE
Better highlighting of the code snippets

### ISSUE THREAD

**Issue by bhupesh-sf** [penge/my-notes#317]
Right now when we add a code snippet it's just a container with no language support. I was thinking if we can add better support for the code highlights which makes the document prettier. For e.g.

[https://highlightjs.org/](url)

**Comment by penge** [comment]
I am not sure about this. On one side, it would look nice, on the other side, is it needed?

I would leave this open to vote.

**Comment by bhupesh-sf** [comment]
Syntax highlighting is always a good option to have a better view OR at least it may match the way Github or similar tools provide the option to add code. Right now it's just a container with no styling.

**Comment by penge** [comment]
I understand it would be nice to have. Also I am not trying to make it tool for everything.

The problem is, that you would like to see styles being applied as you type (which could make it slow).  Having something like https://ace.c9.io.

**Comment by bhupesh-sf** [comment]
Don;t you think if not highlighting but at least markdown styling of code. Also, there can be an option to format everything at once. Because we are not going to write code here but copy-pasting from somewhere else (in most cases)

**Comment by penge** [comment]
Good point. I will think about it. Editing mode seems fine but also addinional step to do as need to recognize when editing, and when not (as that's the moment when styles of highlightjs should be applied).
